src/nbl/asset/IAssetManager.cpp

```diff
@@ -113,6 +113,7 @@ void IAssetManager::initializeMeshTools()
 	m_meshManipulator = core::make_smart_refctd_ptr<CMeshManipulator>();
     m_geometryCreator = core::make_smart_refctd_ptr<CGeometryCreator>(m_meshManipulator.get());
 	m_glslCompiler = core::make_smart_refctd_ptr<IGLSLCompiler>(m_fileSystem.get());
+    m_spirvOptimizer = core::make_smart_refctd_ptr<ISPIRVOptimizer>();
```

Comment:
how much memory does a spirv-optimizer eat up after being used once? (we had a problem where using spirvcross or shaderc meant some 300mb extra mem usage)